src/Knp/DictionaryBundle/Resources/config/legacy/services.yaml

```diff
@@ -0,0 +1,19 @@
+---
+imports:
+  - resource: 'services/debug.yaml'
+  - resource: 'services/factories.yaml'
+  - resource: 'services/faker.yaml'
+  - resource: 'services/forms.yaml'
+  - resource: 'services/templating.yaml'
+  - resource: 'services/transformers.yaml'
+  - resource: 'services/validation.yaml'
+
+services:
+
+  knp_dictionary.dictionary.dictionary_registry:
```

Comment:
It should be the inverse IMHO. Service id should become `Knp\DictionaryBundle\Dictionary\DictionaryRegistry` and we add the alias `knp_dictionary.dictionary.dictionary_registry` to keep BC.